src/polymer/declaration-property-handlers.ts

```diff
@@ -71,11 +72,23 @@ export function declarationPropertyHandlers(
     },
     listeners(node: estree.Node) {
       if (node.type !== 'ObjectExpression') {
+        declaration.warnings.push({
+          code: 'invalid-listeners-declaration',
+          message: '`listeners` property should be an object expression',
+          severity: Severity.ERROR,
+          sourceRange: document.sourceRangeForNode(node)!
+        });
         return;
       }
       for (let prop of node.properties) {
         if (prop.key.type !== 'Literal' || typeof prop.key.value !== 'string' ||
             prop.value.type !== 'Literal' || typeof prop.value.value !== 'string') {
+          declaration.warnings.push({
+            code: 'invalid-listeners-declaration',
+            message: 'handler value should be a string',
+            severity: Severity.ERROR,
+            sourceRange: document.sourceRangeForNode(prop.value)!
+          });
```

Comment:
I'm guessing simple identifiers should still be allowed though, right?
```js
listeners: {
  changed: 'changedHandler'
}
```
How should we treat computed keys? Strictly speaking they should be valid, correct? So should we just skip the entry without a warning or consider a code smell and add a warning with reduced severity? We probably can't extract the event name in that case. Or is there an easy way to evaluate the expression?